BeVolt/App/src/main.c

```diff
@@ -84,14 +85,22 @@ void preliminaryCheck(void){
  */
 void faultCondition(void){
 	Contactor_Off();
+	LED_Off(RUN);
+	
 	while(1){
 		// CAN_SendMessageStatus()
 		if(!Current_IsSafe()){
 			// Toggle Current fault LED
 		}
 		
-		if(!Voltage_IsSafe()){
+		if(Voltage_IsSafe() == OVERVOLTAGE){
+			// Toggle Voltage fault LED
+			LED_On(OVOLT);
+		}
+		
+		if(Voltage_IsSafe() == UNDERVOLTAGE){
```

Comment:
It would be (marginally) faster to call this function once and then switch.

Reply:
Sure, that'll make the grouping of checks consistent as well

Reply:
Can't have enums as the condition statement for switch, maybe `typedef int SafetyStatus` is the way to go